hc/front/tests/test_add_shopify.py

```diff
@@ -0,0 +1,115 @@
+from hc.test import BaseTestCase
+from mock import patch
+
+
+class AddShopifyAlertTestCase(BaseTestCase):
+    """This class contains tests to handle adding checks"""
+
+    def test_it_redirects_add_shopify(self):
+        """test it renders add_shopify """
+
+        self.client.login(username="alice@example.org", password="password")
+        response = self.client.get("/integrations/add_shopify/")
+
+        assert response.status_code == 200
+
+    @patch('hc.front.views.shopify')
+    def test_it_accepts_connection_to_shopify(self, mock):
+        api_key = "84895nfjdufer0n5jnru553jdmfi9"
+
+        password = "d602f072d117438yjfjfjfu9582ce3"
```

Comment:
Set the api_key and password in one place since you're using them in a number of places